code/modules/client/preferences_savefile.dm

```diff
@@ -427,7 +427,7 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 	features["mcolor"]	= sanitize_hexcolor(features["mcolor"], 3, 0)
 	features["ethcolor"]	= copytext_char(features["ethcolor"], 1, 7)
 	features["tail_lizard"]	= sanitize_inlist(features["tail_lizard"], GLOB.tails_list_lizard)
-	features["tail_human"] 	= sanitize_inlist(features["tail_human"], GLOB.tails_list_human, "None")
+	//features["tail_human"] 	= sanitize_inlist(features["tail_human"], GLOB.tails_list_human, "None")
```

Comment:
Why are we commenting out

Reply:
in case some fucker wants to readd tails to prefs for whatever reason

Reply:
This is why git history exists, remove the comments